File: Java/DSA/BubbleSort.java
import java.util.*;

public class BubbleSort{
    public static void bubbleSort(int arr[]){
        for(int turn=0; turn < arr.length-1; turn++){
            for(int j=0; j < arr.length-1-turn; j++){
                if(arr[j] > arr[j+1]){
                    int temp = arr[j];
                    arr[j] = arr[j+1];
                    arr[j+1] = temp;
                }
            }
        }
    }

    public static void  displayArr(int arr[]){
        for(int i=0;i<arr.length;i++){
            System.out.print(arr[i]+" ");
        }
        System.out.println(" ");
    }
    public static void main(String arg[]){
        int num[] = {5,7,3,9,2,1,6,8,0,1};
        displayArr(num);
        bubbleSort(num);
        displayArr(num);


    }
}

File: C/bubbleSort.c
#include <stdio.h>


void bubble_sort(int a[], int n) {
    for(int i=0; i<n-1; i++) {
        for(int j=0; j<(n-i)-1; j++) {
            if(a[j] > a[j+1]) {
                int temp = a[j];
                a[j] = a[j+1];
                a[j+1] = temp;
            }
        }
    }
}

void main() {
    int a[10];
    for(int i=0;i<10;i++) {
        scanf("%d",&a[i]);
    }
    bubble_sort(a,10);
    for(int i=0;i<10;i++) {
        printf("%d ",a[i]);
    }

}